### ISSUE TITLE
Share categories and tags across blogs?

### ISSUE THREAD

**Issue by matt-bailey** [micmania1/silverstripe-blogger#31]
I'm currently planning out a site that may have two blogs on it and I'd like each blog to share the same categories and tags.

I know that out-of-the-box this isn't possible, but do you think it's even remotely plausible?

I suppose another option would be to have nested (sub) categories, so I could have two top-level categories and then all my other categories as sub-categories. Then tags would be shared across all posts anyway. Is this a possibility do you think?

Any thoughts or suggestions on this?

**Comment by sunnysideup** [comment]
We are currently looking at this... Did anything happen with this???? 

Thank you so much ;-)

**Comment by micmania1** [comment]
@sunnysideup nope, but this module is now maintained over at https://github.com/silverstripe/silverstripe-blog

Happy to look over a PR there.

**Comment by sunnysideup** [comment]
Hi Michael,

Thank you... Sorry, I did not pay too much attention last night... We will probably release a tiny add-on module for the blog that does the stuff described above.

**Comment by micmania1** [comment]
Sounds good. If you could post a link here once done I'll close this ticket.
